src/EventStore.Core/Services/VNode/InaugurationManager.cs

```diff
@@ -30,6 +30,7 @@ public class InaugurationManager :
 		private int _currentEpochNumber;
 		private long _replicationCheckpointTarget;
 		private long _indexCheckpointTarget;
+		private readonly IStartupStatusTracker _startupTracker;
```

Comment:
neater to move this up with the other readonly fields

Reply:
Done